src/lib/Components/Bidding/__tests__/BidFlow-tests.tsx

```diff
@@ -1,3 +1,7 @@
+jest.mock("lib/Components/Bidding/Screens/ConfirmBid/PriceSummary", () => ({
```

Comment:
I guess this is a similar comment to the above, but does any dep of ConfirmBid within tests now require that we stub out the component? If so, that seems a bit unfortunate :(